crates/sui-framework/packages/sui-framework/tests/vec_set_tests.move

```diff
@@ -61,4 +61,18 @@ module sui::vec_set_tests {
         assert!(m.size() == 3, 0);
         assert!(m.keys() == &vector[1, 2, 3], 1);
     }
+
```

Comment:
Can you add a test that tries to call `from_keys` with a vector containing duplicate elements to make sure it aborts with `EKeyAlreadyExists`?

Reply:
Ah I'll that for maps too, good catch